src/documentation/transxchange_read.md

```diff
@@ -92,8 +92,17 @@ tram | Tramway | Tramway| Tramway| Tramway
 trolleyBus | Shuttle | Shuttle | Shuttle | Shuttle
 
 ### routes.txt
-A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*/).
-Routes might not be specified at all in the xml --> reconstruct routes after trips?
+A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*).
+
+NTFS field | TransXChange element | Mapping rule/Comment
+--- | --- | ---
+route_id | *Services/Service/ServiceCode*, *StandardService/JourneyPattern/Direction* | This field is prefixed and formed by the concatenation of *Services/Service/ServiceCode* and *StandardService/JourneyPattern/Direction* separated by a `:`. Ex. "\<prefix>:1_58_BC:inbound".
+route_name |  | "[first stop of the first trip] - [last stop of the first trip]" (1)
+direction_type | *StandardService/JourneyPattern/Direction* | The value is set to `inbound` or `clockwise` when the specified value for the direction is `inboundAndOutbound` or `circular`, respectively.
```

Comment:
What if I have 2 `StandardService/JourneyPattern`, the first one having a direction `inbound` and the second one having a direction `inboundAndOutbound`. Is that fine that in the end, we'll have 2 routes, both `inbound` on the same line? The IDs won't collide because they will be `line_id:inbound` and `line_id:inboundAndOutbound`.

Reply:
For now, I would be ok with 2 distinct routes (different ids) with the same direction inbound (or circular).